h/admin.py

```diff
@@ -250,6 +273,31 @@ def groups_index_csv(request):
 
     return {'header': header, 'rows': rows}
 
+class UserDeletionError(Exception):
```

Comment:
Shove this up at the top of the file. Typically I'd try and keep a file ordered something like this:

```
imports
CONSTANTS
Exceptions
Classes
functions
_private_functions
```